pkg/autodiscovery/integration/config.go

```diff
@@ -28,7 +28,7 @@ type JSONMap map[string]interface{}
 
 // Config is a generic container for configuration files
 type Config struct {
-	Name          string   `json:"check_name"`     // the name of the check
```

Comment:
I guess we should change the json serialization as well?

Reply:
I don't think we can, this would lead to backward compatibility issues ?

Reply:
What do you mean? The config serialized json is used out of the agent? 🤔